pkg/run/minio.go

```diff
@@ -54,6 +45,7 @@ const (
 
 // Start - Start the local Minio server
 func (m *MinioServer) Start() error {
+
 	runDir, err := filepath.Abs(m.dir)
```

Comment:
you are creating m.dir but using a hardcoded dir.. you should probably remove one of those

Reply:
fixed. This will currently use a .nitric directory relative to CWD. In most cases users should use `nitric start` command within their project directory (although this is not a requirement).

Reply:
Also removes the mount user permission errors we had with the container version as this runs minio with their account.